packages/js/src/generators/setup-verdaccio/files/config.yml

```diff
@@ -0,0 +1,25 @@
+# path to a directory with all packages
+storage: ../tmp/local-registry/storage
+
+# a list of other known repositories we can talk to
+uplinks:
+  npmjs:
+    url: https://registry.npmjs.org/
+    maxage: 60m
+
+packages:
```

Comment:
I wonder if the behaviour we were seeing regarding needing to be auth'd was due to not having the scoped rule from the nx repo's config. If you add it do we still need `npm adduser`?

Reply:
from https://verdaccio.org/docs/authentication/, it seems like i need to add a line like `//localhost:5555/:_authToken="secretVerdaccioToken"` to `.npmrc`. so i added a command `npm config set //localhost:${options.port}/:_authToken="secretVerdaccioToken"` in the npmSetup and it seems to work without doing `npm adduser`

Reply:
That's awesome! Thanks for looking in to it. Thanks @xiongemi